etc/changes.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+
+This is an incomplete list of changes in the OKAPI interface.
+Please add only changes which are relevant for client developers
+or site admins.
+
+New entries are inserted at the top with EMPTY version number
+and date. Run update_changes.php to insert the version numbers
+and dates.
+
+-->
+<changes>
+    <change commit="93cfc2d9" version="1237" date="2016-03-31" type="enhancement">
+        New methods for &lt;a href="services/logs/images/add.html"&gt;adding&lt;/a&gt;,
```

Comment:
Wouldn't it be easier if we allowed inline XHTML here (`<` instead of `&lt;`)? As we do in XML files for method descriptions?